tests/Extensions/Hosting.Services.Remoting.UnitTests/MockServiceWithDependencies.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.Omex.Extensions.Hosting.Services.Remoting.UnitTests
+{
+	internal class MockServiceWithDependencies : IMockService
+	{
+		private readonly IMockServiceDependency m_dependency;
+
+		public MockServiceWithDependencies(IMockServiceDependency dependency) =>
+			m_dependency = dependency;
+	}
+}
```

Comment:
Add an empty line at the end of the file
```suggestion
}

```